src/gcp/cloudscheduler.ts

```diff
@@ -42,8 +49,13 @@ export async function createOrReplaceJob(schedule: Schedule): Promise<void> {
     if (e.context.response.statusCode !== 409) {
```

Comment:
I think something like this would be more readable:
```
if jobAlreadyExists(e) { // this is status code 409 im assuming?
  // replace job
} else {
  // throw error since we already tried to create job and got a bad error
}
```

Reply:
Refactored this in a slightly different way - I think its cleaner to check for an existing job before we try to create one

Reply:
The benefit of blind create is that you don't have an extra network call to check in the create case. That being said, in most cases update is going to be much more common than create, so it's not a huuuge deal.

Reply:
So, the flip side of having 1 extra call in the create case is that we get to have 1 fewer call in the update case (get + update, instead of create + get + update), since a blind create doesn't tell us whether the existing schedule is identical to the one we are deploying. If we think update is more common, this should reduce the total number of calls over time.